RocketChat/Rocket.Chat
# [IMPROVE] Livechat: Allow external attachments to be resolved

OC-198






## Proposed changes (including videos or screenshots)



Allow external urls to be viewed on livechat.



## Issue(s)

[issue#26511](https://github.com/RocketChat/Rocket.Chat/issues/26511)
## Steps to test or reproduce

paste the below message on a chat, and check if the image appears
```md
![if you can read this on rocket.chat the bug still exists](https://assets.digitalocean.com/articles/alligator/boo.svg)
```
## Further comments

Fixes botpress integration issue
